FitGoal/FitGoal/Setting/SettingsViewController.swift

```diff
@@ -20,5 +28,58 @@ class SettingsViewController: UITableViewController {
             userFitnessLevel: "Intermediate",
             userProfileImage: nil
         )
+        self.tableView.tableFooterView = UIView()
+        self.tableView.register(UserInfoCell.self, forCellReuseIdentifier: UserInfoCell.identifier)
+    }
+    
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return cellInformation.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: UserInfoCell.identifier, for: indexPath)
+        guard let userInfoCell = cell as? UserInfoCell else { fatalError() }
+        userInfoCell.selectionStyle = .none
```

Comment:
Why do you need to set this property here? It looks to me that what you want to do is disable cell selection at all.

Reply:
Are you suggesting moving this to the cell `init`?